pkgs/servers/softether/4.18.nix

```diff
@@ -48,6 +48,6 @@ stdenv.mkDerivation rec {
     homepage = https://www.softether.org/;
     license = licenses.gpl2;
     maintainers = [ maintainers.rick68 ];
-    platforms = platforms.linux;
+    platforms = filter (p: p != "aarch64-linux") platforms.linux;
```

Comment:
The compiler options restricts softether to exactly `x86_64-linux`. so just put `platforms = ['x86_64-linux'];`

Reply:
I was not precise when I said disabling `aarch64`.

Reply:
@Mic92 x86 is supported. They use very rudimentary "configure" script:
`cp src/makefiles/${OS}_${CPU}.mak Makefile`
to pick suitable Makefile.

Reply:
@Mic92 Thanks, aarch64 is disabled now, as I have no aarch64 machines to test. Also they don't support if officially yet: https://www.softether.org/4-docs/1-manual/5._SoftEther_VPN_Bridge_Manual/5.1_Operating_System_Requirements#5.1.2._Support_for_Linux